.circleci/config.yml

```diff
@@ -21,49 +21,62 @@ executors:
           POSTGRES_PASSWORD: postgres
           POSTGRES_HOST_AUTH_METHOD: trust
 jobs:
-
+  # Specify the webpage url (https://dev.dockstore.net or https://staging.dockstore.org or https://dockstore.org)
+  # and stack (dev or staging or prod) and the no auth smoke tests will be run against the corresponding webpage.
   uptime_monitor_no_auth:
+    parameters:
+      stack:
+        type: string
+      webpage_URL:
+        type: string
     working_directory: ~/repo
     docker:
       - image: circleci/buildpack-deps:18.04-browsers
-        auth:
```

Comment:
I removed this auth information as I don't think it impacts the no-auth tests. Unless I'm missing something...?

Reply:
You should keep this. This is using DockerHub credentials to fetch the Docker image. This will decrease the likelihood of us running into a DockerHub rate limiting issue in the future (authorized users have a higher rate limit). I believe CircleCI has some sort of arrangement with DockerHub for now so that pulls from CircleCI don't run into the rate limit, but that may not always be the case.